.github/workflows/redis-tests.yml

```
@@ -0,0 +1,89 @@
+# - ~40 second execution time (~6s cluster startup + ~34s tests)
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'redis/**'
+      - 'online/**'
+      - 'api/**'
+      - '.github/workflows/redis-tests.yml'
+    paths-ignore:
```

actionlint (1.7.9)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)